hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/ConfigurationGenerator.java

```diff
@@ -89,7 +89,6 @@ public Configuration generate(SourceOfRandomness random, GenerationStatus genera
             }
         }
         //ConfigTracker.freshMap();  // --> Comment out if we want to incrementally collect exercised config set.
-        ConfigTracker.clearSetConfigMap();  
```

Comment:
why remove the clear set here?

Reply:
I have another PR and thought this might be kinda overlapped with that PR

Reply:
The current 2 PR are so messy @hwang-pku , can you close #8, and let's use this PR for all bug fixing?

Reply:
Okay, I merged the two PRs together.

Reply:
Thanks a lot!! @hwang-pku